cmd/wasmd/root.go

```diff
@@ -302,3 +305,26 @@ func appExport(
 
 	return wasmApp.ExportAppStateAndValidators(forZeroHeight, jailAllowedAddrs, modulesToExport)
 }
+
+// extendUnsafeResetAllCmd - also clear wasm dir
+func extendUnsafeResetAllCmd(rootCmd *cobra.Command) {
```

Comment:
ah, you copied this over already. I had opened https://github.com/CosmWasm/wasmd/issues/1414 for this.

I was wondering if we want this here or in `x/wasm/cli` ?